### PR TITLE
Added Formula Button (for maths support)

### PR DESCRIPTION
**Pull request by garv-shah**
https://github.com/singerdmx/flutter-quill/issues/898

### CODE REVIEW

File: lib/src/widgets/toolbar.dart [hunk 1]
```diff
@@ -364,6 +365,18 @@ class QuillToolbar extends StatelessWidget implements PreferredSizeWidget {
             webVideoPickImpl: webVideoPickImpl,
             iconTheme: iconTheme,
           ),
+        if (showImageButton)
```
Comment (singerdmx) [inline lib/src/widgets/toolbar.dart:368]: Could you use a new variable?

Comment (garv-shah) [reply lib/src/widgets/toolbar.dart:368]: oh yup, sure

Comment (garv-shah) [reply lib/src/widgets/toolbar.dart]: done!

**Comment by singerdmx** [comment]
@garv-shah https://github.com/singerdmx/flutter-quill/commit/c507e4fccd97ce7c586c2012f882e0887e69d929

**Comment by garv-shah** [comment]
Ah, apologies, it must not have been handling the delta at all.
I believe you said that the attributes for embeds were automatic, it that's not the case would you know where I can handle the delta?

**Comment by singerdmx** [comment]
Delta has to have a line break at the very end

**Comment by singerdmx** [comment]
Don't insert at the end

**Comment by garv-shah** [comment]
oh, whoops! alright, I'll mess around with it some more later, thanks :)

**Pull request merged**